src/views/Dashboard/Customer/Home/RequestTest/SelectServicePage/SelectService.vue

```diff
@@ -133,13 +134,16 @@ export default {
         countRateLab,
         countServiceRate,
         duration,
-        durationType
+        durationType,
+        verifiactionStatus
       }
 
-      this.serviceList.push(service)
+      if (service.verificationStatus === "Verified") {
+        this.serviceList.push(service)
+      }
     }
 
-    if (!this.services.length) {
+    if (this.serviceList === []) {
```

Comment:
Don't compare like this. The previous comparison makes more sense. `!this.servicesList.length`